source/TobiiProFusion.rst

```diff
@@ -0,0 +1,34 @@
+###############################################################################################
+PhysioLab\ :sup:`XR` + Tobii Pro Fusion
+###############################################################################################
+
+This tutorial will demonstrate how to connect the Tobii Pro Fusion Eyetracker to your computer and PhysioLab\ :sup:`XR`.
+The `Tobii Pro Fusion <https://www.tobii.com/products/eye-trackers/screen-based/tobii-pro-fusion>`_ is `Tobii's <https://www.tobii.com/>`_ high-performance eye tracker designed primarily for research purposes, offering precision and flexibility for capturing and analyzing eye movement data.
+This tutorial will be primarily focused on the Windows platform.
+
+1. Turn on the Tobii Pro Fusion.
+2. Launch PhysioLab\ :sup:`XR`.
+3. Go to the **Stream Tab**.
+4. Enter or select the dropdown menu for the **TobiiProFusion**.
+5. Click on the **+Add** button.
+6. Click on the **Start** button to begin streaming data from the Tobii Pro Fusion to PhysioLab\ :sup:`XR`.
+
+.. raw:: html
+
+    <iframe width="560" height="315" src="https://www.youtube.com/embed/j63mmHf7muo"
+    frameborder="0" allowfullscreen></iframe>
+
+7. (Optional) Calibrate your device by entering the address of the Tobii Pro Eye Tracker Manager on your local computer, and clicking on the **Calibration** button.
```

Comment:
For clarity, say "the path to" instead of "the address of". "address" may refer to a network address.